src/Components/TestComponent/index.js

```diff
@@ -0,0 +1,7 @@
+const TestComponent = (props) => {
```

Comment:
dont let this get merged into s1 - can you please remove

Reply:
Removed.